control-plane/pkg/prober/composite_prober_test.go

```diff
@@ -111,7 +111,7 @@ func TestCompositeProber(t *testing.T) {
 			},
 			responseStatusCode:       http.StatusOK,
 			wantStatus:               StatusReady,
-			wantRequeueCountMin:      1,
+			wantRequeueCountMin:      0,
```

Comment:
Why this change?

Reply:
Oh that's my bad, I was messing around with some of the tests and I think I accidentally committed this, I'll revert it when I figure out how to fix the failing test!

Reply:
maybe that's why is failing?

Reply:
Looks like that fixed it locally...